src/include/simeng/models/emulation/Core.hh

```diff
@@ -34,12 +38,16 @@ class Core : public simeng::Core {
   const ArchitecturalRegisterFileSet& getArchitecturalRegisterFileSet()
       const override;
 
+  /** Send a syscall to the system syscall handler. */
+  void sendSyscall(const OS::SyscallInfo syscallInfo) const override;
+
+  /** Communicate the result of a syscall to the core's active exception
+   * handler for post-processing. */
+  void receiveSyscallResult(OS::SyscallResult result) const override;
```

Comment:
Same as [this](https://github.com/UoB-HPC/SimEng/pull/285/files#r1113408565) comment

Reply:
Dealt with similarly to referenced comment

Reply:
The comment in `simeng/Core.hh` reads: "This method receives the result of an initiated syscall and communicates the result to the exception handler for post-processing", however the current comment still says "active" syscall handler.

Reply:
outdated